server/routes/config/curfew.js

```diff
@@ -86,21 +86,23 @@ module.exports = {
           validationMessage: 'Say if the homeowner consents to HDC',
         },
       },
+      {
+        consentHavingSpoken: {
+          validationMessage: 'Say if you were able to speak to the main occupier and if they consented to HDC',
+        },
+      },
       {
         electricity: {
-          dependentOn: 'consent',
```

Comment:
the dependantOn and predicate are not required because the functionality is handled by the bespokeAddressSchema